tests/e2e/TestHelpers/Api.ts

```diff
@@ -303,7 +304,9 @@ async function getProjectContributionDiff(
 
 async function getNotificationsHub(page: Page, handle: string) {
   const data = {
-    notifications: [notification(), notification(), notification()],
+    prevCursor: null,
+    nextCursor: faker.lorem.slug(1),
```

Comment:
@ChrisPenner I'm expecting the JSON shape to optionally include the `prevCursor` and `nextCursor` values